packages/zapp/console/src/components/Entities/EntityInputs.tsx

```diff
@@ -0,0 +1,226 @@
+import {
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+} from '@material-ui/core';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import CheckIcon from '@material-ui/icons/Check';
+import { useLaunchPlans } from 'components/hooks/useLaunchPlans';
+import { formatType, getInputDefintionForLiteralType } from 'components/Launch/LaunchForm/utils';
+import { FilterOperationName } from 'models/AdminEntity/types';
+import { ResourceIdentifier } from 'models/Common/types';
+import { LaunchPlanClosure, LaunchPlanSpec } from 'models/Launch/types';
+import * as React from 'react';
+import t from './strings';
+import { transformLiterals } from '../Literals/helpers';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  header: {
+    marginBottom: theme.spacing(1),
+  },
+  divider: {
+    borderBottom: `1px solid ${theme.palette.divider}`,
+    marginBottom: theme.spacing(1),
+  },
+  rowContainer: {
+    display: 'flex',
+    marginTop: theme.spacing(3),
+  },
+  firstColumnContainer: {
+    width: '60%',
+    marginRight: theme.spacing(3),
+  },
+  secondColumnContainer: {
+    width: '40%',
+  },
+  configs: {
+    listStyleType: 'none',
+    paddingInlineStart: 0,
+  },
+  config: {
+    display: 'flex',
+  },
+  configName: {
+    color: theme.palette.grey[400],
+    fontSize: '14px',
+    marginRight: theme.spacing(2),
+    minWidth: '95px',
+  },
+  configValue: {
+    color: '#333',
+    fontSize: '14px',
+  },
+  headCell: {
+    fontSize: '14px',
```

Comment:
Table text size should come from h4 typography variant
`<Typography className={styles.headCell} variant="h4">`,
as per Figma:
<img width="797" alt="Screen Shot 2022-06-26 at 7 06 16 PM" src="https://user-images.githubusercontent.com/55718143/175825684-d63f9bc6-a1f1-46f1-8cfb-5687b9c9d2fc.png">

Reply:
fixed